BrushedMotorControllerFirmware/include/driver.c

```diff
@@ -8,12 +8,19 @@
 #include "return_codes.h"
 #include "driver.h"
 
+#include "calculations.h"
+
 /// Unit conversion defines
 #define MIN_TO_MS 60000
 #define RPM_TO_MRPM 1000
 
 #define PINS_PER_CHANNEL 2
 
+#define FULL_SPIN_DEGREES (360u * CONFIG_POSITION_CONTROL_MODIFIER)
+#define HALF_SPIN_DEGREES (180u * CONFIG_POSITION_CONTROL_MODIFIER)
```

Comment:
for FULL_SPIN_DEGREES check previous PR. for HALF_SPIN_DEGREES - we can just use FULL_SPIN_DEGREES/2

Reply:
Decided to keep DEGREES_PER_ROTATION and DEGREES_PER_HALF_ROTATION, there is no benefit of neither solution, this one is just preferred

Reply:
DEGREES_PER_ROTATION sgtm